### ISSUE TITLE
Implement host with multiple values

### ISSUE THREAD

**Issue by Nightapes** [exasol/exasol-driver-go#14]
When opening a connection, the driver will randomly choose an address from the specified address range. If the connection fails, the driver will continue to try all other possible addresses. You can also specify an IP range with a comma-separated list.

```
For example:
 
myhost:8563: Single server with name myhost and port 8563. Or single host myhost with multiple IP addresses mapped to it and port 8563.
myhost1,myhost2:8563: Two servers with port 8563.
myhost1..4:8563: Four servers (myhost1, myhost2, myhost3, myhost4) and port 8563.
192.168.6.11..14:8563: Four servers from 192.168.6.11 to 192.168.6.14 and port 8563.
```

**Comment by cyrixsimon** [comment]
How does this range syntax work with hostnames. Which part of the name will be determined as the counter?

The example with myhost1..4 is simple but the hostname could be much more complex. For example giving the following range exasol1.local..exasol3.local - this must be resolved to exasol1.local, exasol2.local, exasol3.local.

**Comment by AnastasiiaSergienko** [comment]
Closed with #17
